### DISCUSSION TITLE
ocr视别，按文档操作测试报错，希望指导一下

### DISCUSSION THREAD

**Discussion by 739184886** [PaddlePaddle/PaddleFormers#1145] [Q&A]
paddlepaddle 2.0
paddlehub 1.8.5
python3.8
centos8 

![image](https://user-images.githubusercontent.com/17465803/103074319-51bbae80-4604-11eb-9a67-9f986c711521.png)

![image](https://user-images.githubusercontent.com/17465803/103074364-6730d880-4604-11eb-9534-7c0d612e0e56.png)

**Comment by chunzhang-hub** [comment]
抱歉才看到讨论，建议之后可以直接提issue问题：https://github.com/PaddlePaddle/PaddleHub/issues，请问咱们这边问题有解决嘛？
